openmrs/apps/registration/app.json

```diff
@@ -22,15 +22,15 @@
         "patientTransferenceInformation": {
                 "title": "patientTransferenceInformation",
                 "translationKey": "REGISTRATION_TRANSFERENCE_INFO_KEY",
-                "attributes": ["Transferred from another HF","Date of transference","Transferred out HF name","Patient stage"],
+                "attributes": ["Transferred from another HF","Date of transference","Transferred out HF name","Patient Status"],
```

Comment:
@amolbch Can you rename it to PATIENT_STATUS?

Reply:
@krishnanspace Updated "Patient Status" to "PATIENT_STATUS"